### ISSUE TITLE
Idea: explain that these numbers don't tell us if our products are accessible, but do tell us if we need to look more closely into something

### ISSUE THREAD

**Issue by carodew** [18F/a11y-metrics#8]
I want to be sure people don't think that if they have zero errors on an automated test that they've created an accessible product.

**Comment by carodew** [comment]
More broadly – add a section about "how to interpret these numbers" – so it could also explain how you can't compare numbers across projects, etc, etc.

**Comment by toolness** [comment]
Good idea--maybe it'd be useful to show the automated metrics in a lighter shade of gray than the manual metrics, to indicate the relative importance of e.g. a manual accessibility audit vs. running `aXe-core` on a site?

**Comment by nkkl** [comment]
Can we delegate this to someone other than @toolness?
